src/containers/Tenant/Schema/SchemaViewer/SchemaViewer.tsx

```diff
@@ -75,22 +90,14 @@ export const SchemaViewer = ({keyColumnIds = [], columns = [], type}: SchemaView
         );
     }
 
-    // Display key columns first
-    const tableData = columns.sort((column) => {
-        if (column.Id && keyColumnIds.includes(column.Id)) {
-            return 1;
-        }
-        return -1;
-    });
-
```

Comment:
`initialSortOrder` should be used instead. Without sorting columns should be displayed as they are in `/json/describe` response